### ISSUE TITLE
When using the find widget and I click on a cell, then I cannot use keyboard navigation.

### ISSUE THREAD

**Issue by barbaravaldez** [microsoft/azuredatastudio#18505]
When using the find widget and I click on a cell. The find widget loses focus and I'm not able to use keyboard navigation.

Version: 1.35.0-rc1
Commit: 560e8974dd3beff152dc3c51548314252a827ad6
Date: 2022-02-19T00:03:12.077Z
VS Code: 1.59.0
Electron: 13.5.0
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin x64 21.3.0
